onnxruntime/core/providers/cpu/math/einsum_utils.cc

```diff
@@ -271,7 +271,7 @@ static std::unique_ptr<Tensor> PairwiseOperandProcess(const Tensor& left,
                       einsum_compute_preprocessor.GetOutputDims(), allocator);
   }
 
-  return std::move(output);
```

Comment:
ran into a warning about a pessimizing move in the manylinux2014 docker container before downgrading the compiler
https://developers.redhat.com/blog/2019/04/12/understanding-when-not-to-stdmove-in-c/